.github/workflows/tests.yaml

```diff
@@ -26,10 +28,17 @@ jobs:
           chmod +x kubectl
           sudo mv kubectl /bin/
 
-      - name: Create Kubernetes cluster
+      - name: Prepare Checkpoints Directory in Host
         run: |
-          kind create cluster
-          kind export kubeconfig
+          sudo mkdir -p ${{ env.CHECKPOINT_DIR }}
+          sudo chmod 700 ${{ env.CHECKPOINT_DIR }}
+          sudo chown $USER ${{ env.CHECKPOINT_DIR }}
```

Comment:
Why do we need to use `chown` here?

Reply:
As you know the operator was having trouble to get the tar files from the `/var/lib/kubelet/checkpoints` directory, I add this to assure that there is no permission issue .. and so it also confirm that we don't need to use `sudo` each time accessing the directory later on the ci run. 

should I remove it ?

Reply:
The checkpoints directory (`/var/lib/kubelet/checkpoints/`) is automatically created by Kubernetes and we don't expect users to use `chown`.

```console
# ls -al /var/lib/kubelet/checkpoints/
total 4
drwx------. 2 root root    6 Jul 27 18:37 .
```
It should be possible to run the tests without this. If there are any problem with permissions, we should figure out how to fix it.